src/tir/transforms/default_gpu_schedule.cc

```diff
@@ -98,24 +98,48 @@ IRModule MarkScheduled(const IRModule& mod) {
                   mod->type_definitions,  // type_definitions
                   mod->import_set_,       // import_set
                   mod->source_map,        // map
-                  mod->attrs);            // attrs);
+                  mod->attrs,             // attrs
+                  mod->global_infos);     // global_infos
+}
+
+bool IsScheduledOnGPU(const BaseFunc& func) {
+  Optional<tvm::Target> target = func->attrs.GetAttr<tvm::Target>(tvm::attr::kTarget);
+  if (target.defined()) {
```

Comment:
Do we need to also check target from context here, i.e., GPU target defined in context but not in the function attribute?